tardis/io/atom_data/simple_atom_data.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+
+import pandas as pd
+
+
+@dataclass
+class SimpleAtomData:
```

Comment:
Is it worthwhile to make this a data class with a single attribute?

Reply:
yep - I'm also not sure as they are all dataframes. I think getting this PR merged without such a dataclasses and then adding PRs for specific classes that do more might be the right way forward.

Reply:
That's fair enough! I can easily remove them. The intent was to provide a baseline structure of classes so that they can be expanded with methods, and this one being likely the base class.

Reply:
I think I would remove them for now until we have a better idea about how we want to proceed.